chatmeleon-server/controllers/user-controller.ts

```diff
@@ -0,0 +1,47 @@
+import prisma from "../libs/prismadb.js"
+import { Request, Response, NextFunction } from "express";
+
+// Define your async route handler function
+
+const getAllUsers = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    // Fetch all users from the Prisma database.
+    const allUsers = await prisma.user.findMany();
+    
+    // Send the list of users as a JSON response.
+    res.send(JSON.stringify(allUsers));
+  } catch (error) {
+    // If there's an error, log it to the console and send an error response.
+    next(error);
+  } finally {
+    // Disconnect from the Prisma database.
+    await prisma.$disconnect();
+  }
+};
```

Comment:
This should be removed since there is not a single context in which this API will be used. Retrieving all users and all their private data should not be done by anyone for multiple security purposes